fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -183,6 +192,116 @@ public DatatableData retrieveDatatable(final String datatable) {
         return datatableData;
     }
 
+    @Override
+    public List<JsonObject> queryDataTable(String datatable, String columnFilter, String valueFilter, String resultColumns) {
+        Arrays.asList(datatable, columnFilter, valueFilter, resultColumns).forEach(SQLInjectionValidator::validateDynamicQuery);
+        List<String> resultColumnNames = Stream.of(resultColumns.split(",")).toList();
```

Comment:
Arrays.asList?

Reply:
any other idea? Stream.of(....)? List.of(....)? what is the question?

Reply:
No question. I'm simply saying to replace Stream.of(..).toList() with an Arrays.asList since you don't use the Stream API anyhow and it's just a waste of resources.

Reply:
Sorry my bad, github showed this comment on another line. I already fixed this.